addons/portal/views/portal_templates.xml

```diff
@@ -48,18 +48,17 @@
         <t t-set="active_lang" t-value="list(filter(lambda lg : lg[0] == lang, languages))[0]"/>
         <t t-set="language_selector_visible" t-value="len(languages) &gt; 1"/>
         <div t-attf-class="js_language_selector #{_div_classes}" t-if="language_selector_visible">
-            <button class="btn btn-sm btn-outline-secondary border-0 dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+            <button t-attf-class="btn btn-sm btn-outline-secondary border-0 dropdown-toggle px-0 #{_btn_class}" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                 <span t-if="not no_text"
                         class="align-middle"
                         t-esc="active_lang[2].split('/').pop()"/>
             </button>
-            <div class="dropdown-menu" role="menu">
+            <div t-attf-class="dropdown-menu #{_dropdown_menu_class}" role="menu">
                 <t t-foreach="languages" t-as="lg">
                     <a t-att-href="url_for(request.httprequest.path + '?' + keep_query(), lang_code=lg[0])"
-                    class="dropdown-item js_change_lang"
-                    t-att-data-url_code="lg[1]">
-                        <span t-if="not no_text"
-                              t-esc="lg[2].split('/').pop()"/>
+                       t-attf-class="dropdown-item js_change_lang  #{'active' if active_lang == lg else none}"
```

Comment:
`none` does not crash? :thinking:

Reply:
It doesn't crash  
used also here: https://github.com/odoo/odoo/blob/00a95c19fdc57247c430ae217520901e23518616/addons/website/static/src/snippets/s_image_gallery/000.xml#L16-L16

but if there will be a potential problem or consistency issue, I can replace with an empty string.

Reply:
... well it is correctly written as `None` where you show me, not `none` :fearful: Don't know how it can work... just use `None` please. Actually, normally we write it like that: `condition and 'my_class'`, this is shorter.